lib/_headers

```diff
@@ -0,0 +1,2 @@
+/*
+  Link: </css/style.css>; rel=preload; as=stylesheet
```

Comment:
Add a newline at the end of code

Reply:
Why?

Reply:
Sorry, there should have been an explanation :)
Newlines on the end of a file is not only a common coding style, but can lead to some issues.
[stackoverflow](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)

Reply:
done!